src/Testcontainers/Containers/Modules/Databases/MySqlTestcontainer.cs

```diff
@@ -38,7 +38,8 @@ public override async Task<ExecResult> ExecScriptAsync(string scriptContent, Can
       await this.CopyFileAsync(tempScriptFile, Encoding.Default.GetBytes(scriptContent), 493, 0, 0, ct)
         .ConfigureAwait(false);
 
-      return await this.ExecAsync(new[] { "mysql", $"--host={this.Hostname}", $"--port={this.ContainerPort}", $"--user={this.Username}", $"--password={this.Password}", this.Database, $"--execute=source {tempScriptFile}" }, ct)
+      // TODO: Shouldn't we always use localhost, if we are inside the container? With 'Hostname' it is failing in Codespaces
```

Comment:
That is an open question. In Codespaces it always showed an IP address which could not be resolved within the container.

Reply:
Good catch. Yes, the command runs inside the container and fail in DinD environments. I suggest we add `MYSQL_ROOT_PASSWORD` first, then take a look at that. `localhost` looks right though.

Reply:
So should i create an issue for that and we can fix it in antother PR?

Reply:
Yes, I would like to update the other commands as well.

Reply:
reverted this commit and created an issue at #632